lib/rubocop/config_loader.rb

```diff
@@ -24,7 +24,7 @@ class << self
 
       attr_accessor :debug, :ignore_parent_exclusion,
                     :disable_pending_cops, :enable_pending_cops
-      attr_writer :default_configuration
+      attr_writer :default_configuration, :project_root
```

Comment:
This is only for test code to be able to reset state, the project_root doesn't really change at runtime. I can alternatively revert this change and use `instance_variable_set` inside the test code to reset it.